src/dash-playlist-loader.js

```diff
@@ -112,18 +113,6 @@ export const parseMasterXml = ({ masterXml, srcUrl, clientOffset, sidxMapping })
   return master;
 };
 
-export const generateSidxKey = (sidxInfo) => {
```

Comment:
This function already existed in mpd-parser, now that we export it, use that one.